__tests__/CoreJS-es6-reflect.test.ts

```diff
@@ -24,21 +24,21 @@ describe('importing ES6 reflect', () => {
     jest.mock('core-js/es6/reflect', () => { throw coreJs2Error }, { virtual: true });
     jest.mock('core-js/es/reflect', () => { throw coreJs3Error }, { virtual: true });
 
-    expect(() => require('../setupJest')).toThrow('core-js es6-reflect not found!');
+    expect(() => require('../src/setupJest')).toThrow('core-js es6-reflect not found!');
```

Comment:
let's move this test into `src/` instead :)

Reply:
I just put it inside `src/`: `src/CoreJS-es6-reflect.test.ts`, better have a look at the folder structure in this branch.

Is this ok? The `__snapshot__` folder also lives inside `src/` now. Or do you think we should separate them somewhat from the source files?

IMO with the relative small amount of source files it is ok to keep them in one folder for now.

Reply:
Hmm why don't we leave test files in src/ __tests__  ? I think it's better and it's a pattern used by each jest package folder too

Reply:
You are right! I argued with being inspired by the facebook/jest setup!

Would you put the `tsconfig.spec.json` for the preset tests inside there too?

Reply:
I think that makes sense 👍